lib/item.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+
```

Comment:
I would recommend adding a directory called `data` or `model` for containing your model files.